### ISSUE TITLE
Suggest features

### ISSUE THREAD

**Issue by deni1023** [GuilleDiazFentanes/AutoClaim-AutoRoll-AutoReact-MudaeBot-2024#3]
Hi, I have a few suggestions.

Firstly, could you make the bot execute the $daily command initially, then check if it receives a green emoji reaction? If it does, proceed with the subsequent commands: $wa, $rolls, and finally, $wa again to claim 20 rolls at once.

For clarity:

1- Type $daily and check for the green emoji.
2- If the green emoji appears, proceed with the sequence: $wa, $rolls, $wa to claim 20 rolls.
- If there's no green emoji, simply execute $wa only.

Secondly, Is it possible to add multiple accounts in the script?
In order not to cause problems, each account plays gradually, but the claim is ready for all accounts.

Additionally, could you add the $dk command?

Thank you!

**Comment by GuilleDiazFentanes** [comment]
In my personal version I use the $tu command to manage all that things, the problem is that languages matter in that case, do you use the english version?
![image](https://github.com/GuilleDiazFentanes/AutoClaim-AutoRoll-AutoReact-MudaeBot-2024/assets/152492889/eea3928c-8831-42f3-ae02-6cb5f185eae4)

**Comment by deni1023** [comment]
Yes, I use the English version.
What about $rolls to reset rolls and take advantage of $daily?

**Comment by GuilleDiazFentanes** [comment]
> Yes, I use the English version.
> What about $rolls to reset rolls and take advantage of $daily?

I have it for the Spanish version but I can try it in the English one, you want daily and rolls ?

**Comment by rahaaatul** [comment]
> I have it for the Spanish version but I can try it in the English one, you want daily and rolls ?

Just changing the command in English will work, no? I can help you with that.

**Comment by deni1023** [comment]
> > Yes, I use the English version.
> > What about $rolls to reset rolls and take advantage of $daily?
> 
> I have it for the Spanish version but I can try it in the English one, you want daily and rolls ?

Yes, in order to benefit from the $daily, I need $rolls to reset rolls and play again
As for the English language, @rahaaatul mentioned that he will help in this matter.

**Comment by tedzify** [comment]
I would also like to suggest to add a function to claim characters equal and above a certain specified kakera amount.

**Comment by deni1023** [comment]
Any updates?

**Comment by rahaaatul** [comment]
@deni1023 Many of your suggestions are implemented in my fork [AutoMudae](https://github.com/rahaaatul/AutoMudae) - check it out!
